lutris/gui/config/common.py

```diff
@@ -544,9 +566,36 @@ def on_custom_image_select(self, _widget, image_type):
         image_filter.add_pixbuf_formats()
         dialog.add_filter(image_filter)
 
+        try:
+            main_file_path = self.game.runner.get_main_file()
+        except AttributeError:
+            main_file_path = None
+
+        path_type = PATH_TYPE.UNKNOWN
+        if image_type.startswith("banner"):
+            path_type = PATH_TYPE.BANNER
+        if image_type.startswith("icon"):
+            path_type = PATH_TYPE.ICON
+
+        def_path = default_path_handler.get(
+            # unfortuantely the original path is not stored
```

Comment:
Misspelling of unfortunately.

Reply:
Yup.  Apparently my spell checker took a holiday, and I didn't realize it.  I'll clean these up this morning.